fcrepo-import/src/main/java/org/fcrepo/importer/ArgParser.java

```diff
@@ -26,13 +27,19 @@
 import org.apache.commons.cli.ParseException;
 
 /**
+ * Command-line arguments parser.
+ *
  * @author awoods
+ * @author escowles
  * @since 2016-08-29
  */
 public class ArgParser implements org.fcrepo.importexport.ArgParser {
 
     final private Options cmdOptions;
 
+    public static final String DEFAULT_RDF_EXT = ".ttl";
```

Comment:
That would be good, because I am getting a log of `Config` javadoc WARNINGs that would need to be addressed otherwise.
```
[WARNING] Javadoc Warnings
[WARNING] /home/awoods/programming/java/apps/fedora/futures/fcrepo-import-export/fcrepo-import/src/main/java/org/fcrepo/importer/Config.java:68: warning: no @param for directory
[WARNING] public void setBinaryDirectory(final String directory) {
```
etc...